plugin/push.configuration.js

```diff
@@ -33,7 +33,8 @@ var checkConfig = function(config) { // jshint ignore:line
     // Controls the sending batch size per interval
     sendBatchSize: Match.Optional(Number),
     // Allow optional keeping notifications in collection
-    keepNotifications: Match.Optional(Boolean)
+    keepNotifications: Match.Optional(Boolean),
+    dbCollectionPrefix: Match.Optional(String)
```

Comment:
add comma at end *(makes future diffs look nicer)*

Maybe just allow setting a `collection` giving the user full control?